devicetypes/smartthings/wemo-switch.src/wemo-switch.groovy

```diff
@@ -16,7 +16,7 @@
  * Date: 2015-10-11
  */
  metadata {
- 	definition (name: "Wemo Switch", namespace: "smartthings", author: "SmartThings") {
+ 	definition (name: "Wemo Switch", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "oic.d.smartplug") {
```

Comment:
I did verify this one. This is the plug it includes the old outlet and the insight outlet :)